src/sonic-frr/patch/series

```diff
@@ -56,3 +56,4 @@
 0094-zebra-fix-yang-data-for-mcast-group.patch
 0095-bgpd-Crash-due-to-usage-of-freed-up-evpn_overlay-att.patch
 0096-bgpd-Notify-all-incoming-outgoing-on-peer-group-noti.patch
+0097-mgmt-Note-that-a-DS-is-locked-or-not-in-output.patch
```

Comment:
Can you please number it as SONiC specific patch if you are not planning to upstream to FRR? If it is upstreamed, can you please provide the FRR community commit details?

Please check the other patches for naming convention.

Reply:
Hi, @dgsudharsan 

Please refer following for FRR commit details, thanks.
```
commit ac505e5f4528f6ce85673e518d1df0c7c10a6ac1
Author: Donald Sharp <sharpd@nvidia.com>
Date:   Fri Jul 11 19:19:15 2025 -0400

    mgmt: Note that a DS is locked or not in output
    
    Modify the `show mgmt datastore ..` command to note
    if the data store is locked and by which session-id.
    
    Signed-off-by: Donald Sharp <sharpd@nvidia.com>
```

Reply:
Thanks @dgsudharsan , updated the patch, thanks.